extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderUserIdentityBox.php

```diff
@@ -13,7 +13,7 @@
 			<?php if ( $canRemoveAvatar ): ?>
 				<span>
 					<img src="<?= $wgBlankImgUrl ?>" class="sprite trash">
-					<a id="UserAvatarRemove" data-name="<?= $user['name']; ?>" href="#" data-confirm="<?= wfMessage( 'user-identity-remove-confirmation' )->escaped(); ?>">
+					<a id="UserAvatarRemove" data-name="<?= Sanitizer::encodeAttribute( $user['name'] ); ?>" href="#" data-confirm="<?= wfMessage( 'user-identity-remove-confirmation' )->escaped(); ?>">
```

Comment:
What do you think about add 'name' to the list we're already escaping [here](https://github.com/Wikia/app/blob/dev/extensions/wikia/UserProfilePageV3/UserIdentityBox.class.php#L124)?